app.py

```diff
@@ -234,6 +234,130 @@ def cleanup_html_data(html_data):
     return (filedata, filedata2, filedata3)
 
 
+def readability_score(final_page):
+    # calculate readability score
+    html = final_page
+    original_soup = BeautifulSoup(html, features="lxml").find("main")
+    original_text = original_soup.get_text()
+    original_text = original_text.replace("..", ".")
+    original_text = original_text.replace(".", ". ")
+    original_text = original_text[: original_text.find("defPreFooter")]
+    original_text = original_text.replace("\n", "")
+    original_text = original_text.replace("\t", "")
+    original_text = original_text.replace("\r", "")
+
+    # get initial readability total_score
+    import textstat
```

Comment:
Imports in the middle of functions can slow down the code, recommended to put at top of the page.